lib/utils/utils.js

```diff
@@ -198,6 +200,25 @@ function isEmptyMethod(node) {
   return node.value.body && node.value.body.body && node.value.body.body.length <= 0;
 }
 
+/**
+ * Gets the name of the module that an identifier was imported from,
+ * if it was imported
+ *
+ * @param {Object} context the context of the ESLint rule
+ * @param {node} node the Idenfifier to find an import for
+ * @returns {string | undefined} The name of the module the identifier was imported from, if it was imported
+ */
+function getImportModule(context, node) {
```

Comment:
I'd love to write some isolated tests for this, but it's kind of tricky because it depends on the ESLint `context` object.

Is there a good way to mock the behavior of that object?

If we want some isolated tests for this, I can build a faux `context` for testing purposes. I've looked around a bunch and haven't found too many examples of helpers for testing utilities used within an ESLint rule like this.

Reply:
Not sure at the moment.

Reply:
I ended up writing a faux `context` test helper and adding tests for the helper (as well as giving it a better name and replacing another utility for which this one super-cedes). I was just about to update this PR with those changes, but I'll do it in a separate PR instead.

Reply:
Sounds good, thank you!